stdlib/types.pyi

```diff
@@ -591,7 +591,7 @@ if sys.version_info >= (3, 9):
             @property
             def __typing_unpacked_tuple_args__(self) -> tuple[Any, ...] | None: ...
 
-        def __getattr__(self, name: str) -> Any: ...  # incomplete
+        def __getattr__(self, name: str) -> Incomplete: ...
```

Comment:
I think this should actually be Any, as GenericAlias delegates attr lookup to its origin.

Reply:
```suggestion
        def __getattr__(self, name: str) -> Any: ...
```

Reply:
Yeah, this one really is inexpressible in the stub